security/pkg/server/ca/server.go

```diff
@@ -168,7 +168,7 @@ func (s *Server) Register(grpcServer *grpc.Server) {
 }
 
 // New creates a new instance of `IstioCAServiceServer`
-func New(ca CertificateAuthority, ttl time.Duration, authenticators []security.Authenticator, client kube.Client) (*Server, error) {
+func New(ca CertificateAuthority, ttl time.Duration, authenticators []security.Authenticator, client kube.Client, filter func(t any) bool) (*Server, error) {
```

Comment:
`filter` should be more readable, to filter what?

Reply:
Its the discovery filter fwiw but I'll rename the variable to be more clear

Reply:
I put a hold if you want to approve and I can rename tomorrow?

Reply:
updated